api/leaderboard/trends_views.py

```
@@ -0,0 +1,421 @@
+                    })
+                cf_user = codeforcesUser.objects.get(username=cf_username)
+                return Response({
+                    "username": cf_username,
+                    "current_rating": cf_user.rating,
+                    "max_rating": cf_user.max_rating,
+                    "source": "live",
+                    "range": range_param,
+                    "history": history,
+                })
+        except Exception:
+            pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)